src/bioregistry/data/collections.json

```diff
@@ -28,5 +28,32 @@
       "ssbd.dataset",
       "ssbd.project"
     ]
+  },
+  "0000002": {
+    "authors": [
+      {
+        "name": "Charles Tapley Hoyt",
+        "orcid": "0000-0003-4423-4370"
+      }
+    ],
+    "contextName": "semweb",
+    "description": "Resources used in the semantic web, based on https://github.com/prefixcommons/biocontext/blob/master/registry/semweb_context.yaml.",
+    "name": "Semantic Web Context",
+    "resources": [
+      "ac",
+      "dc.terms",
```

Comment:
while I do like the idea of sub-namespacing in some situations, it's much more conventional to use `dcterms` as the prefix

Reply:
Addressed in 646c4bd